lib/elixir/test/elixir/kernel/special_forms_test.exs

```diff
@@ -68,4 +68,14 @@ defmodule Kernel.SpecialFormsTest do
       end
     end
   end
+
+  describe "with" do
```

Comment:
We have a "lib/elixir/test/elixir/kernel/warning_test.exs" file, we should most likely move it there. :)

Reply:
I'll move it there. :-)